### DISCUSSION TITLE
Embedding local iframes

### DISCUSSION THREAD

**Discussion by tschaub** [slidevjs/slidev#1919] [Q&A]
I'm trying to use Slidev to create a presentation with slides showing iframes from local sources hosted/built with Vite.

Using Vite without Slidev, I would create an entry `index.html` that looks like this:
```html
<html>
  <head>
    <title>Examples in iframes</title>
  </head>
  <body>
    <section>
      <!-- first slide -->
      <h1>here is the first example</h1>
      <iframe src="./examples/first.html"></iframe>
    </section>

    <section>
      <!-- second slide -->
      <h1>here is the second example</h1>
      <iframe src="./examples/second.html"></iframe>
    </section>
  </body>
</html>
```

Then in my `vite.config.js`, I can create a `build.rollupOptions.input` object referring to the index and each embedded example.

When I've tried to use Slidev to do the same, I can't figure out how to get Vite to serve my embedded examples.  If I put `<iframe src="./examples/first.html"></iframe>` in my `slides.md` file, I get a 404 for `http://localhost:3030/examples/first.html` when running `slidev`.

Is there a way to configure Slidev to allow Vite to serve (and create bundles for) my additional html pages?

**Comment by kermanx** [comment]
https://github.com/slidevjs/slidev/issues/1921#issuecomment-2451556676
